### PR TITLE
feat(tokenUtxoDetails): Added isValid property

### PR DESCRIPTION
**Pull request by christroutner**
This PR satisfies the new behavior described in Issue #84.

### CODE REVIEW

File: src/Util.ts [hunk 1]
```diff
@@ -441,107 +441,104 @@ class Util extends BITBOXUtil {
...
-        // )
-
-        // If the utxo is not SLP, then skip the loop.
-        if (!thisValidation[0] || !thisValidation[0].valid) {
-          outAry[i] = false
-          continue
-        }
+          if (
+            err.message === "Not an OP_RETURN" ||
+            err.message === "Not a SLP OP_RETURN"
```
Comment (SpicyPete) [inline src/Util.ts]: Is it possible to give these errors a code so we don't need to compare directly to the `message` string?  

Error message strings should be modifiable without breaking other code => These comparisons will eventually break from an unrelated change.

Comment (christroutner) [reply src/Util.ts]: With an error code, we'd need to do some sort of lookup in order to give the user an actionable error message. I'm not sure the best way to approach this.

Comment (christroutner) [reply src/Util.ts]: Also, these error codes are being generated within the same Util.ts library (`decodeOpReturn()`). They aren't coming from some external source, so unrelated changes breaking this flow is not a serious concern in my mind. A change would first have to break the unit tests.
